pkg/cmd/server/kubernetes/node.go

```diff
@@ -232,7 +232,7 @@ func (c *NodeConfig) EnsureLocalQuota(nodeConfig configapi.NodeConfig) {
 		// Can't really do type checking or use a constant here as they are not exported:
 		if plugin.CanSupport(emptyDirSpec) {
 			wrapper := emptydir.EmptyDirQuotaPlugin{
-				Wrapped:         plugin,
+				VolumePlugin:    plugin,
```

Comment:
3 upstream commits look to have hit their empty_dir.go.

https://github.com/kubernetes/kubernetes/commit/baa8ac4d6b886e6c5037919a0deaaf5c120750c9
https://github.com/kubernetes/kubernetes/commit/9b6a505f8a28edacb5aae00b951e61806901a59a
https://github.com/kubernetes/kubernetes/commit/542f2dc70802e47564c5d68b88228211f65aa91b

Everything in there looks ok to me.